core/modules/commands/server.js

```diff
@@ -299,13 +299,14 @@ Command.prototype.execute = function() {
 		password: password,
 		pathprefix: pathprefix
 	});
-	this.server.listen(port,host);
+	var nodeServer = this.server.listen(port,host);
 	$tw.utils.log("Serving on " + host + ":" + port,"brown/orange");
 	$tw.utils.log("(press ctrl-C to exit)","red");
 	// Warn if required plugins are missing
 	if(!$tw.wiki.getTiddler("$:/plugins/tiddlywiki/tiddlyweb") || !$tw.wiki.getTiddler("$:/plugins/tiddlywiki/filesystem")) {
 		$tw.utils.warning("Warning: Plugins required for client-server operation (\"tiddlywiki/filesystem\" and \"tiddlywiki/tiddlyweb\") are missing from tiddlywiki.info file");
 	}
+	$tw.hooks.invokeHook('th-server-command-start', this.server, nodeServer);
 	return null;
```

Comment:
Question: What is the point of `return null;`?

Otherwise, is there a scenario where the hook should actually return something which in turn gets returned with the callback?